.gitlab-ci.yml

```diff
@@ -196,7 +198,8 @@ package:
     - if: $DOTNET_PACKAGE_VERSION
       when: on_success
     - if: '$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+(-prerelease)?$/'
-      when: on_success
+      when: manual
+      allow_failure: false
```

Comment:
it's independent from previous job and as it's manual, I assume we could leave `alllow_failure:true`

Reply:
As discussed offline, I don't think so, because that makes the job "fire and forget", whereas we require this stage to pass for subsequent stages to run